src/components/MovieInfo/MovieInfo.js

```diff
@@ -1,16 +1,39 @@
 import React from 'react'
 import './MovieInfo.css'
 
-const MovieInfo = (movie) => {
+const MovieInfo = ({ currentMovieInfo, returnHome }) => {
+  const {
+    id,
+    title,
+    poster_path,
+    backdrop_path,
+    release_date,
+    overview,
+    genres,
+    budget,
+    revenue,
+    runtime,
+    tagline,
+    average_rating
+  } = currentMovieInfo
   return (
     <section className='movie-card'>
-      <img src={movie.backdrop_path} alt='movie backdrop' />
-      <img src={movie.poster_path} alt='movie poster' />
-      <article className='movie-stats'>
-        <h2>{movie.title}</h2>
-        <h3>{movie.rating}</h3>
-        // this is where we will add more info we want to display when we have it from the API fetch
-      </article>
+      <img src={backdrop_path} alt='movie backdrop' className='movie-backdrop' />
+      <div className='movie-display'>
+        <img src={poster_path} alt='movie poster' className='movie-poster' />
+        <article className='movie-stats'>
```

Comment:
Love that we are using all the data...let's see if we can't come up with some sort of star or tomatillo rating system to display the rating for each of these!